cypress/integration/blocks-listing.js

```diff
@@ -66,6 +66,71 @@ if (Cypress.env('API') !== 'guillotina') {
       );
     });
 
+    it('Add Listing Block: sort by effective date', () => {
+      cy.visit('/my-page');
+      cy.waitForResourceToLoad('@navigation');
+      cy.waitForResourceToLoad('@breadcrumbs');
+      cy.waitForResourceToLoad('@actions');
+      cy.waitForResourceToLoad('@types');
+      cy.waitForResourceToLoad('my-page?fullobjects');
+
+      // given a page with two pages
+      cy.createContent('Document', 'page-one', 'Page One', 'my-page');
```

Comment:
The test is green but the functionality is not working so that means the test is not working as it should.
We either have to counter check the "Reversed Order" Button and the order of the results should be the other way around or create another page and make the second one the newest/oldest.

Reply:
You are correct. I did not want to commit a failing test because I want to make sure we can merge it and it passes for now. We can then create a PR that shows the failing test.